geomesa-hbase/geomesa-hbase-spark/src/main/scala/org/locationtech/geomesa/spark/hbase/GeoMesaHBaseInputFormat.scala

```diff
@@ -42,6 +43,17 @@ class GeoMesaHBaseInputFormat extends InputFormat[Text, SimpleFeature] with Lazy
     delegate.setConf(conf)
     // see TableMapReduceUtil.java
     HBaseConfiguration.merge(conf, HBaseConfiguration.create(conf))
+    val auth = conf.get("hbase.security.authentication")
+    auth match{
+      case "kerberos" => {
+        conf.set("hadoop.security.authentication", "Kerberos")
```

Comment:
Just to ask more about this...  is this line changing the value of "hbase.security.authentication" from "kerberos" to "Kerberos"?

Is this a reason for this change?